lib/src/components/NxCollapsibleItems/NxCollapsibleItems.scss

```diff
@@ -112,13 +117,17 @@ $-form-child-left-padding: calc(#{$-trigger-side-padding} + #{$-triangle-box-wid
   width: 100%;
 
   &:hover:not([disabled]) {
-    background-color: var(--nx-swatch-indigo-95);
+    background-color: var(--nx-color-interactive-background-hover);
   }
 
   &:focus {
-    border-color: var(--nx-color-interactive-border-focus);
-    box-shadow: var(--nx-box-shadow-focus);
-    outline: none;
+    @extend %collapsible-items-focus-style;
+  }
+
+  &:active:not([disabled]),
+  &:active:not([disabled]):focus {
+    background-color: var(--nx-swatch-grey-97);
+    border-color: transparent;
```

Comment:
Remove the outline here as well in order to match Kaleido's click state. We also may not need `border-color: transparent` here (if there's no border on `nx-collapsible-items__trigger`, or the border is already transparent), but I could be wrong.

Reply:
Thanks I think that's a good idea: Removed it as part of these two:
https://github.com/sonatype/sonatype-react-shared-components/commit/adbf9c976d5aeae85ad0e19d89530840677a3346
https://github.com/sonatype/sonatype-react-shared-components/commit/caa9f83baf489869fc21d0a649b1d5a01b472039

Reply:
> > 
> 
> > Thanks I think that's a good idea: Removed it as part of these two
> 
> Sorry, I wasn't clear! I also meant remove the `outline: 2px solid var(--nx-color-interactive-border-focus)` that is still applied to the click state.

Ah sorry, i miss-undertood, I think i got what you mean: fixed here: 5f79ed018cb1f0c7eb696eb0de52a29bbba67b79